### DISCUSSION TITLE
Custom download folders

### DISCUSSION THREAD

**Discussion by shrnt** [nianeyna/ao3downloader#134] [Ideas]
Hi, thanks a lot, this has been a life saver!!

It would be nice if I could specify the download folder. I'd like to download all my bookmarks and all my Marked For Later, but I don't want them to be in the same shared folder. I read in another reply you mentioned that you don't wanna download out of the ao3downloader folder for security reasons, so maybe something like `ao3downloader/downloads/bookmarks` or `ao3downloader/downloads/customfoldername`

I guess for now, I could interchange the `ao3downloader/downloads` folder whenever I want to download/update either my Bookmarks or Marked for Later. But it would be convenient if I could specify the destination folder each time before the download starts. Or maybe even be able to define the folders in the `settings.json`

Thanks again!

**Comment by nianeyna** [comment]
Yeah... okayyyyy, fine. You're like the 20th person to request something like this, so I will start thinking of a good way to implement it. The people have spoken, clearly! I'll convert this discussion to an issue, which you can subscribe to in order to see when I've resolved it. As usual, no promises on timeline etc etc, could be tomorrow could be next year.
